l10n/lang-fr.json

```diff
@@ -0,0 +1,92 @@
+{
+  "page": {
+    "translations": {
+      "title": "Logo Interprète",
+      "tl-title": "Logo Interprète",
+      "tl-tests": "Tests",
+      "tl-source": "Source",
+      "tl-reference": "Référence",
+      "start-togetherjs": "Coopérer",
+      "no-canvas": "Votre navigateur ne supporte pas l'élément canvas - Désolé!",
+      "ip-button-run": "Executer",
+      "ip-button-stop": "Arrêter",
+      "ip-button-clear": "Effacer",
+      "logo-ta-single-line.placeholder": "Entrez votre code ici ...",
+      "logo-ta-multi-line.placeholder": "Entrez votre code ici ...",
+      "sb-link-reference": "Référence",
+      "sb-link-text-reference": "le langage Logo",
+      "sb-link-library": "Bibliothèque",
+      "sb-link-text-library": "vos procédures",
+      "sb-link-history": "Histoire",
+      "sb-link-text-history": "Tout ce qui a était fait jusqu'a maintenant",
+      "sb-link-examples": "Exemples",
+      "sb-link-text-examples": "choses amusantes à essayer",
+      "sb-link-extras": "Extras",
+      "sb-link-text-extras": "utilitaires utiles",
+      "sb-link-links": "Liens",
+      "sb-link-text-links": "autres ressources de Logo",
+      "extras-download-library": "Télécharger la bibliothèque",
+      "extras-download-drawing": "Télécharger l'image",
+      "extras-clear-history": "Effacer l'historique",
+      "extras-clear-library": "Effacer la bibliothèque"
+    }
+  },
+  "interpreter": {
+	"messages": {
+      "Can't apply {_PROC_} to special {name:U}": "Ne peut pas appliquer {_PROC_} pour spécial {nom:U",

```

Comment:
Don't localize anything in {}. That is, do not change {name:U} to {nom:U}. The name is a reference to an internal variable.